### PR TITLE
add MAINTAINERs.md file

### PR DESCRIPTION
**Pull request by michelleN**
part of https://github.com/fermyon/spin/issues/2629

### CODE REVIEW

**Review by fibonacci1729 — APPROVED** [review]

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# MAINTAINERS
+
+## Current Maintainers
+
```
Comment (rylev) [inline MAINTAINERS.md:4]: nit: it seems the order is somewhat random which is perhaps the best order (?), but we might want to note that the order is not significant.

**Review by rylev — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (michelleN) [reply MAINTAINERS.md:4]: Good catch. I will alphabetize them by first name like I did for the rest of them!

Comment (michelleN) [reply MAINTAINERS.md:4]: hm. Actually, it really should be name in the first column and username is the second. I will update.

Comment (michelleN) [reply MAINTAINERS.md:4]: Updated. lmk what you think @rylev and I'll update the rest as well.

**Review by lann — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (rylev) [reply MAINTAINERS.md:4]: Looks great! 👍

**Pull request merged**